src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -16,76 +17,88 @@ const ChannelImage = ({
   anonPostNotificationUserInfo = null,
   postMaker = null,
   isCommentExists = false,
-  isOwnSignedPost = false,
-  type = BaseChannelItemTypeProps.ANON_PM
+  type = BaseChannelItemTypeProps.ANON_PM,
+  isAnonymousTab = false
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default value for `type` has been updated. Make sure this change doesn't affect other parts of the code where this function is called without explicitly passing the `type` parameter.
```diff
-  type = BaseChannelItemTypeProps.ANON_PM
+  type = BaseChannelItemTypeProps.ANON_PM,
+  isAnonymousTab = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->